### PR TITLE
Figure indicator merge

### PR DESCRIPTION
**Pull request by emma4you**
Pull request #25957

@alexef @iuliachiriac @anaion  
Please review and merge

### CODE REVIEW

File: hstool/forms.py [hunk 1]
```diff
@@ -205,25 +180,38 @@ def _get_extension(file_name, separator='/'):
...
 
-class FigureForm(ModelForm):
+class FigureIndicatorForm(ModelForm):
+    def __init__(self, *args, **kwargs):
+        super(FigureIndicatorForm, self).__init__(*args, **kwargs)
+        self.fields['sources'].widget.attrs["size"] = 6
+        self.fields['url'].widget.attrs["size"] = 100
+        self.fields['theme'].queryset = (
+            EnvironmentalTheme.objects.filter(is_deleted=False)
+            )
```
Comment (alexef) [inline hstool/forms.py]: whitespace

File: hstool/templates/tool/figures_add.html [hunk 2]
```diff
@@ -21,6 +22,43 @@ <h2 class="page-header">Add/Edit Fact and Figure</h2>
...
+					<a class="btn btn-primary launch-modal" data-toggle="modal"
+							 data-target="#myModal"
+							 data-action="{% url 'modals:add' 'sources'%}"
+							 data-title="Add new Source"
+							 data-backdrop="static">
+						<span class="glyphicon glyphicon-plus"></span>  &nbsp;
+						Add new source
+					</a>
+				</div>
+			</div>
```
Comment (alexef) [inline hstool/templates/tool/figures_add.html]: Indentation.

File: hstool/templates/tool/figures_list.html [hunk 3]
```diff
@@ -38,7 +42,11 @@
 					</td>
 					<td>{{ figure.author_id }}</td>
 					<td>{{ figure.added }}</td>
+                    <td>{{ figure.theme }}</td>
+                    <td>{{ figure.url }}</td>
+                    <td>{{ figure.source }}</td>
 					<td>{{ figure.draft|yesno:"Draft,Published" }}</td>
+                    <td>{{ figure.is_indicator|yesno }}</td>
 					<td>
```
Comment (alexef) [inline hstool/templates/tool/figures_list.html]: idem

File: hstool/tests/test_figures_views.py [hunk 4]
```diff
@@ -21,33 +21,46 @@ def setUp(self):
...
+            resp.pyquery('#objects_listing tbody tr td:eq(9) a').attr('href'),
+            reverse('figures:delete', args=(figure.pk, ))
         )
 
     def test_two_figures(self):
-        figure1 = FigureFactory(file="b", author_id="c")
-        figure2 = FigureFactory(file="bb", author_id="cc")
+        figure1 = FigureFactory()
+        theme = EnvironmentalThemeFactory(title='title')
+        figure2 = FigureFactory(author_id='a2', title='title2', theme=theme, is_indicator='yes', file='file2', url='url2')
```
Comment (alexef) [inline hstool/tests/test_figures_views.py]: long line is long

**Comment by alexef** [comment]
PR lgtm, just check indentation/whitespace.

File: hstool/templates/tool/figures_add.html [hunk 5]
```diff
@@ -2,22 +2,59 @@
...
 				</label>
 				<div class="controls col-sm-10">
-					{% render_field form.title class='form-control' %}
-					{{ form.title.errors }}
+					{% render_field form.name class='form-control' %}
+					{{ form.name.errors }}
+				</div>
+			</div>
+
+            <div class="form-group">
```
Comment (anaion) [inline hstool/templates/tool/figures_add.html]: Please fix indentation

File: hstool/views.py [hunk 6]
```diff
@@ -255,7 +254,7 @@ def dispatch(self, request, *args, **kwargs):
     def get_queryset(self, queryset=None):
         is_admin = self.request.user.has_perm('hstool.config')
         queryset = self.model._default_manager.all()
-
+        print "Here"
```
Comment (anaion) [inline hstool/views.py]: Remove print

**Pull request merged**